compiler/circlechef/tests/string_tensor/test.recipe

```diff
@@ -0,0 +1,31 @@
+operand {
+  name: "ifm"
+  shape { }
+  type: STRING
+}
+operand {
+  name: "constant"
+  type: STRING
+  shape { }
+  filler {
+    tag: "explicit"
+    arg: "Hello"
+  }
+}
+operand {
+  name: "ofm"
+  type: STRING
+  shape { }
+}
+operation {
+  type: "BatchMatMul"
```

Comment:
Is it possible for `BatchMatMul` to run with string type?

Reply:
Actually not possible :) This test is just to test creation of _circle_ model from _recipe_ file.
FYI, files in `circlechef/tests` are not executed.